edit/regexp-tester.js

```diff
@@ -154,24 +154,18 @@ const regexpTester = (() => {
           .split(/(\\+)/)
           .map(s => (s.startsWith('\\') ? $create('code', s) : s)));
       report.appendChild(note);
-      adjustNote(report, note);
+      report.style.paddingBottom = note.offsetHeight + 'px';
     },
   };
 
-  function adjustNote(report, note) {
-    report.style.paddingBottom = note.offsetHeight + 'px';
-  }
-
   function onClick(event) {
-    const a = event.target.closest('a');
+    const a = event.target.closest('a, button');
     if (a) {
       event.preventDefault();
       openURL({
-        url: a.href && a.getAttribute('href') !== '#' && a.href || a.textContent,
+        url: a.href || a.textContent,
```

Comment:
Would this `textContent` contain the button's l18n text?

Reply:
No, the only buttons there are the fakes that contain the tab url.